spec.html

```diff
@@ -36748,6 +36743,20 @@ <h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sec-runtime-semantics-haseitherunicodeflag-abstract-operation" type="abstract operation">
+          <h1>
+            HasEitherUnicodeFlag (
+              _rer_: a RegExp Record,
+            ): a Boolean
+          </h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. If _rer_.[[Unicode]] is *true* or _rer_.[[UnicodeSets]] is *true*, then
+              1. Return *true*.
+            1. Return *false*.
+          </emu-alg>
```

Comment:
You forgot to end your `<emu-clause>` here.

Reply:
```suggestion
          </emu-alg>
        </emu-clause>
```

Reply:
Cheers!